whoc/design_tools/wake_steering_design.py

```diff
@@ -0,0 +1,630 @@
+import numpy as np
+import pandas as pd
+from floris import FlorisModel, UncertainFlorisModel, WindRose, WindTIRose
+from floris.optimization.yaw_optimization.yaw_optimizer_sr import YawOptimizationSR
+from scipy.interpolate import interp1d, RegularGridInterpolator
+
+
+def build_simple_wake_steering_lookup_table(
+    fmodel: FlorisModel,
+    wd_resolution: float = 5,
+    wd_min: float = 0,
+    wd_max: float = 360,
```

Comment:
Totally up to you, but I would do 360.0 to insist on it being a float.  But looking around I think either is fine